book.rb

```diff
@@ -0,0 +1,15 @@
+require './rentals'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+
+  def add_rentals=(person, date)
+    Rental.new(self, person, date)
+  end
```

Comment:
- [ ] The method `add_rentals=` serves as a setter typically because of the `=` sign. The previous reviewer asked that you fix that by having only `add_rental`.

- [ ] You should also be mindful of the order of the parameters here. The order of the parameters in the Rental constructor is this, `person`, `book` `date`. You need to follow the same order here:

  ```
  def add_rental(person, date)
    Rental.new(person, self, date)
  end
  ```